deploy/infrastructure/dev/us-east-2/kms.tf

```diff
@@ -35,6 +35,7 @@ data "aws_iam_policy_document" "kms_sti" {
         "arn:aws:iam::407967248065:user/marco",
         "arn:aws:iam::407967248065:user/gammazero",
         "arn:aws:iam::407967248065:user/will.scott",
+        "arn:aws:iam::407967248065:user/kylehuntsman",
```

Comment:
should this be a separate pr for auditing? i'm okay just lumping together for now though

Reply:
I made it a separate commit for that purpose; happy to separate the PR too

Reply:
https://github.com/filecoin-project/storetheindex/pull/326